lib/server/src/server.rs

```diff
@@ -25,6 +26,10 @@ use self::trow_server::*;
 use crate::server::trow_server::registry_server::Registry;
 
 
+use prometheus::{TextEncoder, Encoder};
+
+use crate::statics::{FREE_SPACE, AVAILABLE_SPACE, TOTAL_SPACE, TOTAL_MANIFEST_REQUESTS, TOTAL_BLOB_REQUESTS};
```

Comment:
as below, rename statics to metrics. But what about changing this to `use crate::metrics;` and not importing the individual metrics? Then below you would write `metrics::FREE_SPACE.inc()` etc, which I think makes it a bit clearer what is going on.